### PR TITLE
Finalize Request #7853

### PR DESCRIPTION
**Pull request by unl-creq**
Updating bulletin section.

### CODE REVIEW

**Comment by tlemburg** [comment]
Looks good

File: data/2015/majors/Agricultural & Environmental Sciences Communication.xhtml [hunk 1]
```diff
@@ -68,61 +65,77 @@
...
+
+<p class="requirement-sec-1">Required  Communication Courses: 24</p>
+<p class="requirement-sec-2">JOMC 101 Principles of Mass Media 3 </p>
+<p class="requirement-sec-2">JOMC 160 Visual Production: Photography &amp; Videography 3 </p>
+<p class="requirement-sec-2">JOMC 161 Visual Design Fundamentals: Graphic Design &amp; Web 3 </p>
+<p class="requirement-sec-2">JOUR 201 Editing I 3 </p>
+<p class="requirement-sec-2">JOUR 202 Reporting I <span class="requirement-ital">(if not taken as ACE 1)</span> 3</p>
+<p class="requirement-sec-2">ADPR 251 Introduction to Advertising &amp; Public Relations 3 </p>
+
+<p class="requirement-sec-2"><span class="requirement-ital">Choose two courses </span><span class="requirement-ital"> from the following</span>: 6</p>
```
Comment (kabel) [inline data/2015/majors/Agricultural & Environmental Sciences Communication.xhtml]: Fix the span balance.

File: data/2015/majors/Agricultural & Environmental Sciences Communication.xhtml [hunk 2]
```diff
@@ -68,61 +65,77 @@
...
-<p class="requirement-sec-1">Option 2: 18</p>
-<p class="requirement-sec-2">Required courses: 6</p>
-<p class="requirement-sec-3">ADPR 251 Introduction to Advertising &amp; Public Relations 3</p>
-<p class="requirement-sec-3">ADPR 283 Strategy Development for Advertising &amp; Public Relations 3</p>
-<p class="requirement-sec-2">Group 1 – <span class="requirement-ital">Select 6 credits from the following courses:</span> 6</p>
+<p class="requirement-sec-1">Option 2: 18 </p>
+<p class="requirement-sec-2">Required courses: 3</p>
+
+<p class="requirement-sec-3">ADPR 283 Strategy Development for Advertising &amp; Public Relations (3 cr)</p>
+<p class="requirement-sec-2">Group 1 – Issues Courses <span class="requirement-ital"></span><span class="requirement-ital"></span> 6 </p>
```
Comment (kabel) [inline data/2015/majors/Agricultural & Environmental Sciences Communication.xhtml]: Remove the empty span.

File: data/2015/majors/Agricultural & Environmental Sciences Communication.xhtml [hunk 3]
```diff
@@ -183,30 +208,43 @@
 <p class="requirement-sec-3">WATS 281 Introduction to Water Science (GEOG 281/NRES 281) (3 cr)</p>
 <p class="requirement-sec-3">WATS 354 Soil Conservation &amp; Watershed Management (prereqs) (MSYM 354/SOIL 354) (3 cr)</p>
 <p class="requirement-sec-3">WATS 361 Soils, Environment &amp; Water Quality (AGRO 361/GEOL 361/SOIL 361) (3 cr)</p>
-<p class="requirement-sec-2">Group 2 – <span class="requirement-ital">Select 9 credits from the following courses:</span> 9</p>
+<p class="requirement-sec-2">Group 2 – Issues Courses<span class="requirement-ital"></span><span class="requirement-ital"></span> 6 </p>
```
Comment (kabel) [inline data/2015/majors/Agricultural & Environmental Sciences Communication.xhtml]: Remove the empty spans.

File: data/2015/majors/Agricultural & Environmental Sciences Communication.xhtml [hunk 4]
```diff
@@ -183,30 +208,43 @@
...
 <p class="requirement-sec-3">ENVR 249 Individual &amp; Cultural Perspectives on the Environment (3 cr)</p>
 <p class="requirement-sec-3">ENVR 319 Environmental Engagement &amp; the Community (2 cr)</p>
 <p class="requirement-sec-3">FDST 280 Contemporary Issues in Food Science (2 cr)</p>
 <p class="requirement-sec-3">NRES 104 Climate in Crisis (3 cr)</p>
 <p class="requirement-sec-3">NRES 323 Natural Resources Policy (3 cr)</p>
-<p class="requirement-sec-1">Total College Core Requirements 40-43</p>
-<p class="requirement-sec-1">Total Agricultural and Environmental Sciences Communication Program Requirements 41</p>
-<p class="requirement-sec-1">CASNR Course Requirement 18</p>
-<p class="requirement-sec-1">Program Emphasis Requirement:</p>
+<p class="requirement-sec-2">Group 3 - Communication Courses<span class="requirement-ital"></span> 6 </p>
```
Comment (kabel) [inline data/2015/majors/Agricultural & Environmental Sciences Communication.xhtml]: Remove the empty span.

**Pull request merged**